### PR TITLE
Instantiate AdminKernel

### PR DESCRIPTION
**Pull request by ga-devfront**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Instantiate the appropriate kernel based on the environment
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | [Fixe issue #35029](https://github.com/PrestaShop/PrestaShop/issues/35029)
| Sponsor company   | @PrestaShopCorp
| How to test?      |

### CODE REVIEW

**Review by Hlavtox — COMMENTED** [review]
Why it should instantiate front kernel in dev mode? 🤔 Am I missing something? :D

**Comment by ga-devfront** [comment]
> Why it should instantiate front kernel in dev mode? 🤔 Am I missing something? :D

I'm sorry I started investigating the ticket and I didn't fully understand the problem. The PR is in WIP I'm trying to resolve the problem as quickly as possible.

File: classes/UpgradeTools/SymfonyAdapter.php [hunk 1]
```diff
@@ -59,23 +61,44 @@ public function runSchemaUpgradeCommand()
...
      */
-    public function initAppKernel()
+    public function initKernel()
     {
         global $kernel;
         if (!$kernel instanceof \AppKernel) {
-            require_once _PS_ROOT_DIR_ . '/app/AppKernel.php';
             $env = (true == _PS_MODE_DEV_) ? 'dev' : 'prod';
-            $kernel = new \AppKernel($env, _PS_MODE_DEV_);
-            if (method_exists($kernel, 'loadClassCache')) { // This method has been deleted in Symfony 4.x
```
Comment (M0rgan01) [inline classes/UpgradeTools/SymfonyAdapter.php:73]: The comment is useful for understanding, I think we can keep it

**Review by jolelievre — APPROVED** [review]

**Review by Hlavtox — APPROVED** [review]

**Review by M0rgan01 — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Comment by AureRita** [comment]
Hi @ga-devfront 

Thank you for your PR, Currently, I tried to  test your PR. The installation going well until I want to go in my admin panel 

This is the error showed : 
image

Tested on 8.1.5 and 8.0.5 

Do you know how we can improve this ? 

Waiting for feedback

**Review by AureRita — APPROVED** [review]
Seen with @MatShir, 

Because this PR correct the initial issue and let us discover another one, we'll validate this PR and create a new issue, So this PR is QA :heavy_check_mark: 

Thank you @ga-devfront for your correction !

**Pull request merged**